src/components/v5/common/ActionSidebar/partials/MultiSigSidebar/MultiSigSidebar.tsx

```diff
@@ -14,6 +15,17 @@ const displayName = 'v5.common.ActionSidebar.partials.MultiSig';
 interface MultiSigSidebarProps {
   transactionId: string;
 }
+
+function hasMultiSig(action: ColonyAction): action is Omit<
+  ColonyAction,
+  'multiSigData' | 'multiSigId'
+> & {
```

Comment:
Good type guard 👌 Slight nitpick would be to rename this as `isMultiSig` because it makes it read better:
```js
if (isMultiSig(action)) {
  console.log("Multi-Sig ID: ", action.multiSigId);
  console.log("Multi-Sig data: ", action.multiSigData.somethingSomething);
}
```

It would be a good idea to declare this function in a more "global" utils folder and export it so other files can utilise it